.github/CODEOWNERS

```diff
@@ -185,11 +185,18 @@ pkgs/development/python-modules/buildcatrust/ @ajs124 @lukegb @mweinelt
 # Licenses
 /lib/licenses.nix @alyssais
 
-# Qt / KDE
-/pkgs/applications/kde @ttuegel
-/pkgs/desktops/plasma-5 @ttuegel
-/pkgs/development/libraries/kde-frameworks @ttuegel
-/pkgs/development/libraries/qt-5 @ttuegel
+# Qt
+/pkgs/development/libraries/qt-5 @NixOS/qt-kde
```

Comment:
github isn't happy about this

> Unknown owner on line 189: make sure the team @NixOS/qt-kde exists, is publicly visible, and has write access to the repository
> 
> …gs/development/libraries/qt-5 @NixOS/qt-kde

Reply:
I noticed, but I think it's just github being weird. The team name is clickable even in your comment.